tests/testthat/test-github.R

```diff
@@ -55,3 +55,41 @@ test_that("github_DESCRIPTION", {
     github_DESCRIPTION("r-lib", "remotes", ref = "1.0.0", use_curl = TRUE),
     desc)
 })
+
+test_that("github_error", {
+  mockery::stub(
+    github_error,
+    "curl::parse_headers_list",
+    list(`x-ratelimit-remaining` = 0, `x-ratelimit-reset` = "1539962178"))
+
+  # Test without the TRAVIS envvar set
+  withr::with_envvar(c(TRAVIS = NA), {
+    err <- github_error(list(headers = "", status_code = "304", content = charToRaw('{"message": "foobar"}')))
+    expect_equal(conditionMessage(err),
```

Comment:
I think this would be slightly cleaner if it used `expect_known_output(conditionMessage(err), print = TRUE, ...)` since it makes it more clear that it's a regression test. Otherwise, LGTM

Reply:
Ok, good suggestion, I have done so